e2e-tests/local-testnet-debug.sh

```diff
@@ -1,4 +1,4 @@
-#!/bin/bash
+#! /bin/bash
```

Comment:
Why is this change done here and below? Is it needed?
Seeing other scripts under `tests/e2e/testnet-scripts` in the `interchain-security` repository, there's no space.

Reply:
Will revert that, good point